huntflow_api_client/entities/accounts.py

```diff
@@ -0,0 +1,32 @@
+from huntflow_api_client.entities.base import BaseEntity
+from huntflow_api_client.models.response.accounts import (
+    MeResponse,
+    OrganizationInfoResponse,
+    OrganizationsListResponse,
+)
+
+
+class Account(BaseEntity):
+    async def get_current_user(self) -> MeResponse:
+        path = "/me"
+        response = await self._api.request(
+            "GET",
+            path,
+        )
```

Comment:
Why did you split it into two lines? And in other methods too. `path` variable seems redundant

`response = await self._api.request("GET", "/me")`